hack/update-translations.sh

```diff
@@ -83,8 +83,7 @@ if [[ "${generate_pot}" == "true" ]]; then
   # shellcheck disable=SC2046
   go-xgettext -k=i18n.T $(grep -lr "i18n.T" "${KUBECTL_FILES[@]}" | grep -vE "${KUBECTL_IGNORE_FILES_REGEX}") > tmp.pot
   perl -pi -e 's/CHARSET/UTF-8/' tmp.pot
-  perl -pi -e 's/\\\(/\\\\\(/g' tmp.pot
-  perl -pi -e 's/\\\)/\\\\\)/g' tmp.pot
+  perl -pi -e 's/\\(?!n"\n|")/\\\\/g' tmp.pot
```

Comment:
this seems reasonable

To summarize: we are trying to preserve all of our `\` symbols that might be used in for example escapes for shell commands and we want  to print them to the user. To preserve them we have to escape them with `\`. But xgettext puts its own `\` symbols around `"` and converts newlines into `\n` so we have to skip these two cases.